package/conda-recipe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "quarto-cli" %}
+{% set version = "dev" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  path: ../..
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - deno
```

Comment:
Our source code is definitely sensitive to these version requirements:

- Deno must be v1.22.0 (higher versions will have compile errors)
- Pandoc must be 2.19 (previous versions will fail w/ unsupported command line options)

Generally speaking our Deno and Pandoc requirements will remain _very specific_ as both projects frequently introduce breaking changes.

Reply:
yes, I'm wary of that. I would like to get some kind of test suite running as part of this package build. I don't have the older versions of deno/deno-dom built on conda-forge yet. I will get them added.